espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.kt

```diff
@@ -66,6 +70,9 @@ class EspressoServerRunnerTest {
             Assume.assumeTrue(true)
             return
         }
+
+        disableCaptureScreenshotOnFailureByEspressoLib()
```

Comment:
lets hardcode it for now. Later we could make it configurable with capabilities if needed

Reply:
Ok